features/guidance/turn-angles.feature

```diff
@@ -680,6 +682,58 @@ Feature: Simple Turns
             | j,o       | Kapweg,Kapweg,Kapweg | depart,continue uturn,arrive |
             | a,i       | Kurt,Kurt,Kurt       | depart,continue uturn,arrive |
 
+    #http://www.openstreetmap.org/#map=18/52.56251/13.32650
```

Comment:
same scenario as above, but now without the additional short segment before the turn